### ISSUE TITLE
Formatting a VHDL file with Windows (CRLF) line endings adds additional blank lines after each formatting

### ISSUE THREAD

**Issue by wrightsg** [InES-HPMM/emacs-vhdl-formatter-vscode#1]
Running Visual Studio Code on a Windows 10 machine, remotely connected to a Ubuntu 18.04 machine.

The issue only occurs if the VHDL file has Windows (CRLF) line endings.
Formatting works as expected if the VHDL file has Linux line endings.

If I open a VHDL file with Windows line endings in `emacs` and run VHDL mode beautify, the issue does not occur.

How to reproduce:
1) Convert any VHDL file to Windows line endings using `unix2dos` command on Linux
2) Format the VHDL file in Visual Studio Code using the plugin
3) Additional blank lines are added between each line after each formatting

Expected behavior: same as if running VHDL mode beautify from `emacs` directly.

**Comment by delafthi** [comment]
Thank you for the issue, I will look into it.

**Comment by delafthi** [comment]
This issue seems to be harder to solve than I first thought. I couldn't exactly locate the issue.

Initially, I thought that emacs would not recognize the dos encoding. However, when I test the my emacs call in the terminal, the output looks fine (though, I'm not sure if the crlf line endings remain when I copy and paste the dos format file into the terminal. Nonetheless, inserting `"\r\n"` and printing it out also yields the correct result in the terminal). So, my current guess is that the output on stdout is converted to the dos encoding, while it already is dos encoded (maybe `\r` and `\n\ get replaced by `\r\n`). However, I still need further investigation to proof that statement.

Additionally, this issue isn't specific to vscode. I also encountered it in neovim, where I also implemented the same functionality (in [null-ls.nvim](https://github.com/jose-elias-alvarez/null-ls.nvim/blob/main/doc/BUILTINS.md#emacs_vhdl_mode)).
